sign_node/signer.py

```diff
@@ -111,11 +111,19 @@ def on_sync_request(self, queue, message):
             password = self.__password_db.get_password(
                 payload['key_id']
             )
+            sig_type = payload.get('sig_type', 'detach-sign')
+            if not sig_type:
+                sig_type = 'detach-sign'
```

Comment:
it looks like unreachable block because we get value by `sig_type` key even if it's not in dict

Reply:
Rewrote as Stepan suggested, it looks more consistent